test/cpp/util/test_credentials_provider.cc

```diff
@@ -153,23 +140,16 @@ class DefaultCredentialsProvider : public CredentialsProvider {
                                                             test_server1_cert};
         ssl_opts.pem_key_cert_pairs.push_back(pkcp);
       }
-      ssl_opts.min_tls_version = TLS1_2;
-      ssl_opts.max_tls_version = TLS1_2;
-      return SslServerCredentials(ssl_opts);
-    } else if (type == grpc::testing::kTls13CredentialsType) {
-      SslServerCredentialsOptions ssl_opts;
-      ssl_opts.pem_root_certs = "";
-      if (!custom_server_key_.empty() && !custom_server_cert_.empty()) {
-        SslServerCredentialsOptions::PemKeyCertPair pkcp = {
-            custom_server_key_, custom_server_cert_};
-        ssl_opts.pem_key_cert_pairs.push_back(pkcp);
-      } else {
-        SslServerCredentialsOptions::PemKeyCertPair pkcp = {test_server1_key,
-                                                            test_server1_cert};
-        ssl_opts.pem_key_cert_pairs.push_back(pkcp);
+      if (type == grpc::testing::kTlsCredentialsType) {
```

Comment:
Is this first if statement necessary? Won't this occur by default?

Reply:
On second thought: please change this to a switch statement. In the `kTlsCredentialsType` case, you can just `break`. In the default case, we can fail.

Reply:
I think it may be cleaner just to remove it as you initially suggested. I put the first case initially because I thought being explicit may be useful, but it seems much cleaner just removing it. I'm not sure if a switch statement would be necessary and failing seems excessive since we check the condition right above in the if condition. WDYT?